__tests__/app.test.js

```diff
@@ -32,3 +32,39 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  describe("/api/articles/:article_id", () => {
+    test("GET: 200 - responds with a specific article by searching its artticle_id", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(typeof body.article.author).toBe("string");
+          expect(typeof body.article.title).toBe("string");
+          expect(typeof body.article.article_id).toBe("number");
+          expect(typeof body.article.body).toBe("string");
+          expect(typeof body.article.topic).toBe("string");
+          expect(typeof body.article.created_at).toBe("string");
+          expect(typeof body.article.votes).toBe("number");
+          expect(typeof body.article.article_img_url).toBe("string");
+        });
+    });
```

Comment:
As we are dealing with just 1 article object, we can be really specific in our tests and test the values from the object as well rather than just the data types 😄 You can even use `.toEqual` for this so you have only one assertion